src/Microsoft.VisualStudio.Editors/PropPages/DebugPropPage.vb

```diff
@@ -163,6 +166,39 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             Return True
         End Function
 
+        Private Function SetAuthenticationMode(control As Control, prop As PropertyDescriptor, value As Object) As Boolean
+
+            Dim mode As AuthenticationMode = VSLangProj165.AuthenticationMode.AuthenticationMode_Windows
+
+            Try
+                mode = CType(value, AuthenticationMode)
+            Catch
+            End Try
+
+            Select Case mode
+                Case VSLangProj165.AuthenticationMode.AuthenticationMode_None
+                    AuthenticationMode.SelectedItem = My.Resources.Strings.NoAuthentication
+                Case Else
+                    AuthenticationMode.SelectedItem = My.Resources.Strings.WindowsAuthentication
```

Comment:
This repo is past the localization cut-off for the 16.5 release, so we generally can't take any resource changes at this time. The change will either need to be held for 16.6 or you'll need to get an exception approved before we can merge this in for 16.5.

Reply:
Can you forward me documents to start exception process? We really need this in 16.5 so that customers who develop for new Windows 10X devices have some option for debugging apps on them.